content/source/docs/cloud/integrations/service-now/developer-reference.html.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: "cloud"
+page_title: "Developer Reference - ServiceNow Service Catalog Integration -
+Terraform Cloud and Terraform Enterprise"
+description: |-
+  Developer reference for the ServiceNow Service Catalog Integration.
+---
+
+## Terraform Variables and ServiceNow Variable Sets
+
+ServiceNow has the concept of a Variable Set which is a collection of ServiceNow Variables that can be referenced in a Flow from a Service Catalog item. The Terraform Integration codebase can create [Terraform Variables and Terraform Environment Variables](../../workspaces/variables.html) via the API using the `tf_variable.createVariablesFromSet()` function.
+
+This function looks for variables following these conventions:
+
+ServiceNow Variable Name | Terraform Cloud Variable
+--|--
+`tf_var_VARIABLE_NAME` | Terraform Variable: `VARIABLE_NAME`
+`tf_env_ENV_NAME` | Environment Variable: `ENV_NAME`
+`sensitive_tf_var_VARIABLE_NAME` | Sensitive Terraform Variable (Write Only): `VARIABLE_NAME`
+`sensitive_tf_env_ENV_NAME` | Sensitive Environment Variable (Write Only): `ENV_NAME`
+
+This function takes the ServiceNow Variable Set and Terraform Workspace ID. It will loop through the given variable set collection and create any Terraform variables or Terraform environment variables.
```

Comment:
```suggestion
This function takes the ServiceNow Variable Set and Terraform Cloud workspace ID. It will loop through the given variable set collection and create any necessary Terraform variables or environment variables in the workspace.
```